consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/PayloadIdentifier.java

```diff
@@ -40,6 +40,7 @@ public static PayloadIdentifier forPayloadParams(final Hash parentHash, final Lo
     return new PayloadIdentifier(((long) parentHash.toHexString().hashCode()) ^ timestamp);
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
If this is removed, I will get the warning again. 
Is the right approach to removing the getValue() in interface Quantity?

Reply:
Since it is a public API (potentially used by third-party plugins), we don't want to _remove_ that method from Quantity just yet. But we can certainly remove usage of it in Besu code as you are doing.

Reply:
And it's fine to use locally while you're narrowing things down, but we don't want to add the `@SuppressWarnings` annotation to main Besu code.

Reply:
Gotcha, I will remove it.